react/src/demo/example-data/deckgl-map.json

```diff
@@ -32,6 +32,7 @@
         {
           "@@type": "ColormapLayer",
           "id": "colormap-layer",
+          "name": "Surface map",
```

Comment:
Suggest "Property map" as default name for this layer.

Suggest also having default names for each layer, which will be used if not specified explicitly.

Reply:
Resolved